packages/base-types/src/trace/completion/completion-start-trace.ts

```diff
@@ -0,0 +1,43 @@
+import { BaseTraceFrame, TraceType } from '@base-types/node/utils';
+
+export interface CompletionStartTrace extends BaseTraceFrame<BaseCompletionStartTracePayload> {
+  type: TraceType.COMPLETION_START;
+}
+
+export interface CompletionStartTraceSpeak extends CompletionStartTrace {
+  payload: CompletionStartTraceSpeakPayload;
+}
+
+export interface CompletionStartTraceText extends CompletionStartTrace {
+  payload: CompletionStartTraceTextPayload;
+}
+
+export type CompletionStartTracePayload = CompletionStartTraceSpeakPayload | CompletionStartTraceTextPayload;
+
+export interface CompletionStartTraceSpeakPayload extends BaseCompletionStartTracePayload {
+  type: TraceType.SPEAK;
+  voice?: string;
+}
+
+export interface CompletionStartTraceTextPayload extends BaseCompletionStartTracePayload {
+  type: TraceType.TEXT;
+  delay?: number;
+}
+
+export interface BaseCompletionStartTracePayload {
+  type: string;
+  completion: string;
+  tokens?: {
+    model: string;
```

Comment:
Do we have one?

Reply:
There is one called `AIModel` in `@voiceflow/dtos`, though if it's in DTOs, it's not worth to import because it will create a circular reference structure between `@voiceflow/dtos` and `@voiceflow/base-types`.